src/components/FourOhFour/FourOhFour.jsx

```diff
@@ -39,11 +38,18 @@ const Styled = styled.div`
   }
 `;
 
+const imageList = ['sad', 'bawl', 'cry', 'charm-cry'];
```

Comment:
I'd label these as the names of the heroes featured to make it more readable. . . unless these are the names of the original files?

Reply:
It looks like cry and charm-cry are both undying, is there another crying gif of a different hero?

Reply:
Yeah those are the original names of the emoticons, I found them here
https://dota2.gamepedia.com/Emoticons
But these gifs here are 32x32px, not 64x64 so I did some photoshop to make them the same as `sad`. I only found these images, there are couple of different options like WR `facepalm` but not exactly the same so I didn't put.

Reply:
In that case, I'd suggest only keeping one of the undying images so there's an equal chance of each hero.

Reply:
Updated. I keep the more pretty version of undying and remove another. Also tested, still working ok.